bindings/odin/clay-odin/clay.odin

```diff
@@ -31,8 +31,8 @@ Dimensions :: struct {
 
 Arena :: struct {
     label:          String,
-    nextAllocation: c.uint64_t,
-    capacity:       c.uint64_t,
```

Comment:
Thanks for all the work here - appreciate it!
I'd just like to get a bit of clarification here from the discord, because I'm a bit confused about this section in the docs:

https://odin-lang.org/news/binding-to-c/#types

<img width="861" alt="Screenshot 2024-09-01 at 10 38 14 AM" src="https://github.com/user-attachments/assets/25f6a34d-dc66-4901-9e06-44912a727a24">

Reply:
I have just asked and got clarification - there's no functional difference for sized types < register width - so yep this change is great! Get to get rid of a whole bunch of casts which is awesome